environments/common/inventory/group_vars/all/defaults.yml

```diff
@@ -18,11 +18,19 @@ prometheus_address: "{{ hostvars[groups['prometheus'].0].api_address }}"
 
 ############################# bootstrap: local user configuration #########################
 
-appliances_local_users_default: "{{ [appliances_local_users_podman] if appliances_local_users_podman_enable else [] + appliances_local_users_extra }}"
+appliances_local_users_default: "{{ (appliances_local_users_podman if appliances_local_users_podman_enable else []) + [appliances_local_users_ansible_user] + appliances_local_users_extra }}"
 # Overide this to add extra users whilst keeping the defaults.
 appliances_local_users_extra: []
 appliances_local_users: "{{ appliances_local_users_default + appliances_local_users_extra }}"
 
+appliances_local_users_ansible_user_name: "{{ ansible_user_id }}"
+appliances_local_users_ansible_user_home: /var/lib/{{ appliances_local_users_ansible_user_name }}
+appliances_local_users_ansible_user:
```

Comment:
Noticed that this is slightly inconsistent with `appliances_local_users_podman` which is a list. The reason why `appliances_local_users_podman` is a list is because I imagined one user per podman service i.e kibana, opendistro, filebeat and I was thinking they'd all be added to that list.